field_encrypt.module

```diff
@@ -100,11 +105,12 @@ function field_encrypt_form_alter(&$form, FormStateInterface $form_state, $form_
  */
 function field_encrypt_form_field_add_form_builder($entity_type, \Drupal\field\Entity\FieldStorageConfig $field_storage_config, &$form, \Drupal\Core\Form\FormStateInterface $form_state) {
   $field_encryption_settings = $form_state->getValue('field_encrypt');
+  $field_encryption_settings['encrypt'] = (bool) $field_encryption_settings['encrypt'];
   $form_encryption = $field_encryption_settings['encrypt'];
   $original_encryption = $field_storage_config->getThirdPartySettings('field_encrypt');
 
   // If the form has the value, we set it.
-  if ($form_encryption === 1) {
+  if ($form_encryption) {
```

Comment:
Nit picks, this variable seems excessive - shouldn't this just be `$field_encryption_settings['encrypt']`?